src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -146,6 +148,21 @@ void fillInnerParts() {
 
         CommandBox commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
+
+        // Displays the first person in the list if exists onto the PersonDetailView
+        if (!logic.getFilteredPersonList().isEmpty()) {
+            personDetailView = new PersonDetailView(logic.getFilteredPersonList().get(0));
+            personDetailViewPlaceholder.getChildren().setAll(personDetailView.getRoot());
+        }
+    }
+
+    /**
+     * Updates the person details view when a person is selected from the list.
+     * @param person The selected person to display in the detail view.
+     */
+    private void displayPersonDetails(Person person) {
+        personDetailView = new PersonDetailView(person);
```

Comment:
Perhaps instead of creating a new `PersonDetailView` object here, we can just call `update`

Reply:
We still need to display the `PersonDetailView` on the `MainWindow` for the UI to be updated. Calling `update` only will only change the data on the `PersonDetailView`